### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by mkraljt** [jtablesaw/tablesaw#1201] [General]
Hi,
Would it be possible to give a new release. The Arrow Stream writer that is on master is very useful.

Thanks

**Comment by lwhite1** [comment]
i don't think master will be ready for a long time. It might be easier to
backport the stream writer to the Java 8 branch.

**Comment by benmccann** [comment]
Sorry, this library hasn't had a release in awhile. I'll cut a new release. Will track it in https://github.com/jtablesaw/tablesaw/issues/1251
